src/commands/check.js

```diff
@@ -218,6 +220,47 @@ export const handler = catchAsyncError(async opts => {
         isUpdateFinished = true;
         break;
 
+      case 'commit':
+        updatedModules.push(outdatedModule);
+        setModuleVersion(name, to, packageJson);
+        delete config.ignore[name];
+
+        // Showing the list of modules that are going to be updated
+        console.log(
+          `\n${strong('These packages will be updated:')}\n\n` +
+          createUpdatedModulesTable(updatedModules) +
+          '\n'
+        );
+
+        const {indent} = detectIndent(packageSource);
+        writeFileSync(
+          packageFile,
+          // Adding newline to the end of file
+          `${JSON.stringify(packageJson, null, indent)}\n`
+        );
+
+        try {
+          if (existsSync('yarn.lock')) {
```

Comment:
We don't need to make this check every time, right? It can be done only once.

Reply:
True